src/containers/location/providers/ProviderMap.js

```diff
@@ -3,7 +3,7 @@ import React, { useEffect, useMemo, useReducer } from 'react';
 
 import isFunction from 'lodash/isFunction';
 import ReactMapboxGl, { ZoomControl } from 'react-mapbox-gl';
-import { List, Map, isImmutable } from 'immutable';
+import { get, List, Map } from 'immutable';
```

Comment:
i think should be sorted this way?

`import { List, Map, get } from 'immutable';`